javascript/ql/src/semmle/javascript/security/dataflow/ShellCommandInjectionFromEnvironmentCustomizations.qll

```diff
@@ -0,0 +1,62 @@
+/**
+ * Provides default sources, sinks and sanitisers for reasoning about
+ * command-injection vulnerabilities, as well as extension points for
+ * adding your own.
+ */
+
+import javascript
+import semmle.javascript.security.dataflow.TaintedPathCustomizations
```

Comment:
:clap:

Reply:
Just checking: this is not a sarcastic clap because it looks like a copy/paste mistake, right?

I did consider moving the path resolution bits out of `TaintedPathCustomizations.qll` and into a more query-neutral library..

Reply:
Not at all! It's a brilliant vindication of our design.